mbox-operator/roles/mbs-frontend/tasks/main.yml

```
@@ -48,3 +48,19 @@
         src: "{{ mbs_frontend_dir.path }}/mbs-configmap.yaml"
 
 - include_tasks: cert.yml
+
+- block:
+    - name: ensure mbs-backend deployment
+      template:
+        src: deployment.yaml.j2
+        dest: "{{ mbs_dir.path }}/deployment.yaml"
+    - k8s:
+        src: "{{ mbs_dir.path }}/deployment.yaml"
```

And another one here
```suggestion
        src: "{{ mbs_frontend_dir.path }}/deployment.yaml"
```